src/main/java/com/example/java_pro_homework_7/Pet/Dog.java

```diff
@@ -0,0 +1,48 @@
+package com.example.java_pro_homework_7.Pet;
+
+public class Dog extends Animals {
+
+    public Dog(String name, String color, String breed) {
```

Comment:
Class' members should have a specific order:

constants
static variables
instance variables
constructors
getters + setters
public methods
private methods
Please change this class' members order according to the info in this comment.